Desktop/utilities/imgschemehandler.cpp

```diff
@@ -12,6 +13,8 @@ using namespace Modules;
 ImgSchemeHandler::ImgSchemeHandler(QObject *parent)
 	: QWebEngineUrlSchemeHandler{parent}
 {
+    std::cout << "ImgSchemeHandler for QtWebengine" << std::endl;
```

Comment:
now the application stoped here to start...any ideas to go deubg here? @JorisGoosen 

or build a release with debug symbols version?

Reply:
in logs we can see:
```
Setting special options for software rendering (aka safe graphics).
Making new argument list for Application startup: JASP.exe --logToFile --disable-gpu
Starting JASP 0.19 from commit 0e4d561737223e223956950a6c9f6d5a5e1b3f6d and branch lockfiletesting
QtWebEngineQuick initialized
Application initialized
PlotSchemeHandler for QtWebengine
```